optimizely/event/event_processor.py

```diff
@@ -54,12 +56,14 @@ def __init__(self,
                 logger,
                 default_start=False,
                 event_queue=None,
+                notification_center=None,
```

Comment:
add at the last. One general rule, if you are adding any new arg in the existing code, then don't add it in the middle, add it at the last.

Reply:
Noted.